core/src/test/scala/org/apache/spark/deploy/worker/WorkerSuite.scala

```diff
@@ -60,7 +60,7 @@ class WorkerSuite extends SparkFunSuite with Matchers with BeforeAndAfter {
     val securityMgr = new SecurityManager(conf)
     val rpcEnv = RpcEnv.create("test", "localhost", 12345, conf, securityMgr)
     _worker = new Worker(rpcEnv, 50000, 20, 1234 * 5, Array.fill(1)(RpcAddress("1.2.3.4", 1234)),
-      "Worker", "/tmp", conf, securityMgr, shuffleServiceSupplier)
+      "Worker", "/tmp", conf, securityMgr, None, Map.empty, shuffleServiceSupplier)
```

Comment:
if its easier to do with the next jira for assigments that is fine too

Reply:
We need to get Master in to test wether Worker sets up the resources correctly, since Worker could not start up without a living Master. Maybe, we could cover this in the following JIRA task, which would sharing resource info between Master and Worker.

Reply:
sounds fine